apps/web/components/Proposals.tsx

```diff
@@ -210,44 +211,49 @@ export function Proposals({
     useTransactionNotification(allocateData);
 
   useEffect(() => {
-    triggerRenderProposals();
+    // triggerRenderProposals();
+    runIsMemberQuery();
   }, [txConfirmationHash]);
 
   useEffect(() => {
     updateTransactionStatus(allocateStatus);
   }, [allocateStatus]);
 
   const submit = async () => {
-    const encodedData = getEncodedProposals(inputs, stakedFilters);
+    const proposalsDifferencesArr = getProposalsInputsDifferences(
+      inputs,
+      stakedFilters,
+    );
+    const encodedData = encodeFunctionParams(cvStrategyABI, "supportProposal", [
+      proposalsDifferencesArr,
+    ]);
     const poolId = Number(strategy.poolId);
     writeAllocate({
       args: [BigInt(poolId), encodedData as AddressType],
     });
   };
 
-  const getEncodedProposals = (
+  // this calculations breaks when dealing with < 50 wei
+  const getProposalsInputsDifferences = (
     inputData: ProposalInputItem[],
     currentData: ProposalInputItem[],
   ) => {
     const resultArr: [number, BigInt][] = [];
     inputData.forEach((input) => {
       let row: [number, bigint] | undefined = undefined;
-      if (input.value > 0) row = [Number(input.id), BigInt(input.value)];
+      if (input.value > 0)
+        row = [Number(input.id), BigInt(Math.round(input.value))];
       currentData.forEach((current) => {
         if (input.id === current.id) {
-          const dif = BigInt(input.value - current.value);
+          const dif = BigInt(Math.round(input.value) - current.value);
```

Comment:
what reasons for the rounding?

Reply:
because input value can be a float number and i can't use BigInt constructor with a float value.
if we are dealing with < 100 wei and a slider input of 100 places we will get a bigger denominator than the nominator so in this case we need to be able to handle that.
This is an ugly solution but the only i came out with, do you have any suggestions?

Reply:
just wonder why round the `input` but not the `current` also

Reply:
because input can be a float but current is member stakes for each proposal that comes from subgraph

Reply:
current value comes from subgraph its member stakes, so no need to round it, but i just found out that i am not using the parentesis correctly there